### PR TITLE
Disable travis ci

### PR DESCRIPTION
**Pull request by coreydaley**
- Remove .travis.yml file
- Update make verify to run make dep to install dependencies

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @bparees 

Ben, Can you also disable Travis CI for this repository?

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .DEFAULT_GOAL := help
 
-verify: ## Run core verification. Example: make verify
+verify:  dep ##Run core verification. Example: make verify
```
Comment (bparees) [inline Makefile]: ?

Comment (coreydaley) [reply Makefile]: That runs make dep when you run make verify

**Comment by bparees** [comment]
> Ben, Can you also disable Travis CI for this repository?
done

File: Makefile (hunk 1 above)
Comment (bparees) [reply Makefile]: yeah....why would we do that?  i wouldn't expect a "Verify" to change my content.

(also put it on its own line, like we have elsewhere, separate from the comment)

Comment (coreydaley) [reply Makefile]: We need to install the dependencies for the python script that does the import?

Comment (bparees) [reply Makefile]: i don't need to do that if i have them installed.

that sounds like a pre-req step, not a verify step.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .DEFAULT_GOAL := help
 
-verify: ## Run core verification. Example: make verify
+verify: ##Run core verification. Example: make verify
```
Comment (bparees) [inline Makefile]: drop this change

Comment (coreydaley) [reply Makefile]: updated in current push

**Pull request merged**